app/routes/events/view/edit/sessions-speakers.js

```diff
@@ -14,7 +14,19 @@ export default class SessionsSpeakersRoute extends Route.extend(EventWizardMixin
     const tracksPromise = data.event.get('tracks');
     const microlocationsPromise = data.event.get('microlocations');
     const sessionTypesPromise = data.event.get('sessionTypes');
-    const speakersCallPromise = this.getOrCreate(data.event, 'speakersCall', 'speakers-call');
+    const speakersCallPromise = new RSVP.Promise(async resolve => {
+      await data.event
```

Comment:
Don't await here, it's useless and will block the loading. Move these statements in another function and await in that function. No await should be there in model function